bims/static/js/router.js

```diff
@@ -17,6 +17,16 @@ define(['backbone', 'views/olmap', 'shared'], function (Backbone, olmap, Shared)
                     }
                 }
             });
+            $.ajax({
```

Comment:
i see you put ajax call on router.
and also for collector.

Maybe could you put for now in app?
https://github.com/kartoza/django-bims/blob/develop/bims/static/js/app.js#L54

router is just for router, so we need to make it clean from other intentions